### ISSUE TITLE
Simplify documentation

### ISSUE THREAD

**Issue by khillion** [motleystate/moonstone#65]
## Description
Some parts are redundant in the documentation when the listing is done, both in API docs and User guide.

We need to have only one listing, in API documentation.

**Comment by AgnesBaud** [comment]
I didn't see this issue before opening the #88 PR, but I was thinking of keeping the listing in available classes and have a more detailed listing (with listing of the methods) in API documentation, like what is done in the Filtering and Normalization API documentation. What do you think?

**Comment by khillion** [comment]
Yes sure! The important thing in my opinion is to make sure we do not have exactly the same things at two different places in the documentation.
